### ISSUE TITLE
Typos and Editorials

### ISSUE THREAD

**Issue by pasetti** [pnp-software/fwprofile#4]
This ticket was initially raised as Mantis Issue 759:
- In section 4.5 of the FW Profile Definition document: "exaple" --> "Example"
- In section 4.2 of the FW Profile Definition document: in constraint C7, use "transition trigger" instead of "transition command" for consistency with terminology used in constraint C5.
- In table 4, "Notification" is mis-spelt twice.
- In section 5.1 of the FW Profile User Manual: "an application notifies
  a RT Container through the following function call: FwRtExecute(rtDesc)" --> an application notifies a RT Container through the following function call: FwRtNotify(rtDesc)".
- In the FW Profile Definition Document, change: PP-DF-COR-00001 to: PP-DF-COR-0001

**Comment by oppm** [comment]
1) UserManual: There is a reference included in RtContainerProperties.tex (\ref{sec:supportForReusability}) which cannot be resolved. I do not know to which section this reference should point.
2) When I generate the pdfs, the chapter numbers are omitted in contrast to the existing pdf. Is this intended, i.e. have you changed the layout of the docs ?
3) Shall I add the tex files also or simply update the PDF and leave the tex files in SVN ?

**Comment by cechticky** [comment]
1) and 2) I'd need to be at a computer. I'm responding fom my mobile phone now. 3) It's about whether we as a company decide to make a documentation sources as open source. It requires internal discussion. Personally, why not if we find a suitable open source licence.

**Comment by pasetti** [comment]
1) The file RtContainerProperties.tex is imported into several documents. Unfortunately, the reference to \ref{sec:supportForReusability} is only correct in one of these documents. Hence, there is no choice but to completely remove the sentence with the reference. The sentence to be deleted is the following one: 

> Thus, the container properties represent invariant properties of a RT Container in the sense of section \ref{sec:supportForReusability}

2) This is a bug in the Latex distribution associated to Ubuntu 16.04. The solution is to add the following text in the header of the document just after `\usepackage[compact]{titlesec}`:

```
% Start ---- Fix for bug in issue 2.10.1 of titlesec package
\usepackage{etoolbox}

\makeatletter
\patchcmd{\ttlh@hang}{\parindent\z@}{\parindent\z@\leavevmode}{}{}
\patchcmd{\ttlh@hang}{\noindent}{}{}{}
\makeatother
% End ---- Fix for bug in issue 2.10.1 of titlesec package
```

You will find an example of the above in any of our Cheops documents.

**Comment by pasetti** [comment]
The typos and editorial have been fixed and the changes have been pulled into the master branch.
